xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/XWikiSyntaxEscapeWikiPrinter.java

```diff
@@ -236,15 +236,15 @@ public void printVerbatimContent(String verbatimContent)
             }
         }
 
-        if (subVerbatimStack.size() > 0) {
+        if (!subVerbatimStack.isEmpty()) {
```

Comment:
Be careful you change the logic! :(

Reply:
But it is correct , when the size is not empty it should be greater than 0 (as it cannot be negative)

Reply:
Indeed you're right sorry, went too fast in my review :)